From a081eaeb728433efc67c81f656e27a103b737e71 Mon Sep 17 00:00:00 2001
From: "Jan.Hannig" <jan.hannig@irisgmbh.de>
Date: Wed, 11 May 2022 14:07:09 +0200
Subject: [PATCH] [RDPHOEN-958] Remove CONFIG_CMD_BOOTEFI for
 imx8mp_irma6r2_defconfig

As the function "efi_set_bootdev" called within this compiler switch
causes the error msg "**Bad device specification mmc 2#linuxboot_a **".

Signed-off-by: Jan.Hannig <jan.hannig@irisgmbh.de>
---
 configs/imx8mp_irma6r2_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/imx8mp_irma6r2_defconfig b/configs/imx8mp_irma6r2_defconfig
index e4333589b4..1dc7fa61b3 100644
--- a/configs/imx8mp_irma6r2_defconfig
+++ b/configs/imx8mp_irma6r2_defconfig
@@ -40,6 +40,7 @@ CONFIG_SPL_POWER_SUPPORT=y
 CONFIG_HUSH_PARSER=y
 CONFIG_SYS_PROMPT="u-boot=> "
 # CONFIG_BOOTM_NETBSD is not set
+# CONFIG_CMD_BOOTEFI is not set
 # CONFIG_CMD_EXPORTENV is not set
 # CONFIG_CMD_IMPORTENV is not set
 CONFIG_CMD_ERASEENV=y
-- 
2.20.1

